hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/invalidation/RepairingHandler.java

```diff
@@ -213,4 +213,14 @@ public String toString() {
                 + ", localUuid='" + localUuid + '\''
                 + '}';
     }
+
+    public void initUuid(int partitionID, UUID partitionUuid) {
+        MetaDataContainer metaData = getMetaDataContainer(partitionID);
+        metaData.setUuid(partitionUuid);
+    }
+
+    public void initSequence(Integer partitionID, Long partitionSequence) {
```

Comment:
`int partitionID` in `initUuid()`, `Integer partitionID` here. Should be the same?